docs/features/pdb-compilation-options.md

```diff
@@ -257,20 +255,28 @@ Example:
-
-See [compiler options](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/listed-alphabetically) documentation
+#### Shared Options for CSharp and Visual Basic
 
 | PDB Key                | Format                                  | Default   | Description  |
 | ---------------------- | --------------------------------------- | --------- | ------------ |
-| language               | `CSharp`                                | required  | Language name. |
-| language-version       | `[0-9]+(\.[0-9]+)?`                     | required  | [langversion](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option) |
+| language               | `CSharp|Visual Basic`                   | required  | Language name. |
 | compiler-version       | [SemVer2](https://semver.org/spec/v2.0.0.html) string | required | Full version with SHA |
 | runtime-version        | [SemVer2](https://semver.org/spec/v2.0.0.html) string | required | [runtime version](#runtime-version) |
-| optimization           | `(debug|debug-plus|release)`            | `debug`   | [optimization](#optimization) |
+| source-file-count      | int32                                   | required    | Count of files in the document table that are source files |
+| optimization           | `(debug|debug-plus|release|release-debug-plus)` | `'debug'` | [optimization](#optimization) |
 | portability-policy     | `(0|1|2|3)`                             | `0`       | [portability policy](#portability-policy) |
 | default-encoding       | string                                  | none      | [file encoding](#file-encoding) |
 | fallback-encoding      | string                                  | none      | [file encoding](#file-encoding) |
+| output-kind            | string                                  | require   | The value passed to `/target` |
+| platform               | string                                  | require   | The value passed to `/platform` |
+
+#### Options For CSharp
+
+See [compiler options](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/listed-alphabetically) documentation
+
+| PDB Key                | Format                                  | Default   | Description  |
+| ---------------------- | --------------------------------------- | --------- | ------------ |
+| language-version       | `[0-9]+(\.[0-9]+)?`                     | required  | [langversion](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option) |
 | define                 | `,`-separated identifier list           | empty     | [define](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/define-compiler-option) |
 | checked                | `(True|False)`                          | `False`   | [checked](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/checked-compiler-option) |
```

Comment:
```suggestion
| checked                | `(True\|False)`                          | `False`   | [checked](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/checked-compiler-option) |
```

Reply:
Ha, you and I were doing this in parallel. Pushed up a new commit that should resolve all of these.